ref_builder/resources.py

```diff
@@ -123,6 +108,25 @@ def convert_name(cls: "RepoIsolate", value: Any) -> IsolateName | None:
 
         raise ValueError(f"Invalid type for name: {type(value)}")
 
+
+class RepoIsolate(IsolateModel):
```

Comment:
```suggestion
class Isolate(IsolateModel):
```

Reply:
I think `RepoOTU` and `RepoIsolate` should be changed in a different PR, on account of how many files those two changes would touch.